test/itkScancoImageIOTest3.cxx

```diff
@@ -79,6 +79,18 @@ itkScancoImageIOTest3(int argc, char * argv[])
   ITK_TEST_EXPECT_TRUE(itk::Math::FloatAlmostEqual(scancoIO->GetDataRange()[1], 11662.0, 6, 1e-3));
   std::cout << "MuScaling: \t\t" << scancoIO->GetMuScaling() << std::endl;
   ITK_TEST_EXPECT_TRUE(itk::Math::FloatAlmostEqual(scancoIO->GetMuScaling(), 8192.0, 6, 1e-3));
+  std::cout << "MuWater: \t\t" << scancoIO->GetMuWater() << std::endl;
+  ITK_TEST_EXPECT_TRUE(itk::Math::FloatAlmostEqual(scancoIO->GetMuWater(), 0.24090, 6, 1e-3));
```

Comment:
In test2, `scancoIO->GetMuWater()` was `0.7033`. I assume this different value comes from the image. Are different `MuWater` normal? Or is this variation more for testing?

Reply:
This value (0.7033) was taken directly from the test ISQ image. The MuWater values will vary with different scanner calibrations.